anilist.py

```diff
@@ -155,45 +155,45 @@ def _make_request(query: str, variables: dict):
         print(f'429: {response}. Waiting {timeout_seconds} seconds...', file=sys.stderr)
         time.sleep(timeout_seconds)
 
+    time.sleep(0.7) #This should _theoretically_ mean we never hit the 429 again.
```

Comment:
Note: I was still getting the 429 when I had to run uncached data. 

Only getting about 30-40 calls instead of the advertised 90.